pulseapi/settings.py

```diff
@@ -12,22 +12,20 @@
 
 import os
 import dj_database_url
+import environ
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+app = environ.Path(__file__) - 1
+root = app - 1
 
-import environ
+BASE_DIR = root()
+APP_DIR = app()
 
 environ.Env.read_env(os.path.join(BASE_DIR,'.env'))
 env = environ.Env(
     DEBUG=(bool, False),
     SSL_PROTECTION=(bool, False),
+    USE_S3=(bool, True),
```

Comment:
If we're leaving SSL Protection off by default foe dev purposes, shouln't we do the same for S3?